conftest.py

```diff
@@ -1,4 +1,6 @@
 def pytest_addoption(parser):
+    parser.addoption('--large-only', action='store_true', dest="large_only",
```

Comment:
Add a `--large-only` option so that we can exclusively run tests marked as large in `./travis/run-large-python-tests.sh`